install/kubernetes/tetragon/templates/_container_tetragon.tpl

```diff
@@ -38,6 +38,10 @@
       name: export-logs
     - mountPath: "/procRoot"
       name: host-proc
+{{- if ne .Values.tetragon.cri.socketHostPath "" }}
```

Comment:
if you want you can simplify with, or can you use `enabled` that you define for this field?
```suggestion
{{- if .Values.tetragon.cri.socketHostPath }}
```

Reply:
Makes sense. Used: `{{- if and (.Values.tetragon.cri.enabled) (.Values.tetragon.cri.socketHostPath) }}`